movie/load.go

```diff
@@ -23,7 +23,7 @@ var movieTypes = []string{
 	"Action",
 	"Adventure",
 	"Animation",
-	"Children",
+	"Children's",
```

Comment:
It also needs to be modified at the same time. Maybe we need to define a set of constants.

https://github.com/Mini256/tidb-dataset/blob/cbcc9150b061088db6e4521ccc6d49677bdf8123/movie/load.go#L109